src/main.ts

```diff
@@ -281,6 +327,7 @@ export class Purchases {
    * @returns a Promise for the customer info after the purchase is completed successfully.
    * @throws {@link PurchasesError} if there is an error while performing the purchase. If the {@link PurchasesError.errorCode} is {@link ErrorCode.UserCancelledError}, the user cancelled the purchase.
    */
+  @requiresInitialization
```

Comment:
Note that currently only this method is decorated, since it's the only one that right nows requires "initialization".